src/linear_algebra/eigenspace.lean

```diff
@@ -34,8 +34,7 @@ notation for it.
 
 ## References
 
-* [Sheldon Axler, *Linear Algebra Done Right*,
-  https://zhangyk8.github.io/teaching/file_spring2018/linear_algebra_done_right.pdf][axler2015]
+* Sheldon Axler, *Linear Algebra Done Right* [axler2015]
```

Comment:
```suggestion
* [Sheldon Axler, *Linear Algebra Done Right*][axler2015]
```
This seems to be the pattern used elsewhere, e.g. https://github.com/leanprover-community/mathlib/blob/80a9e4f/src/set_theory/pgame.lean#L100